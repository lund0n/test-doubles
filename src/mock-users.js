export const ARTHUR_DENT = {
  id: 42,
  name: 'Arthur Dent',
  username: 'panickedBrit',
  email: 'DentedFord42@yahoo.com',
}

export const FORD_PREFECT = {
  id: 99,
  name: 'Ford Prefect',
  username: 'vogon-hater',
  email: 'TowelWearer1@gmail.com',
}
