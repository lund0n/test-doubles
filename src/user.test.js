import React from 'react'
import renderer from 'react-test-renderer'
import User from './user'
import { ARTHUR_DENT } from './mock-users'

test('renders correctly', () => {
  const component = renderer.create(<User {...ARTHUR_DENT} />).toJSON()
  expect(component).toMatchSnapshot()
})
