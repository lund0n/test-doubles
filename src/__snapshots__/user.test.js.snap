// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<dl>
  <dt>
    ID:
  </dt>
  <dd>
    42
  </dd>
  <dt>
    Name:
  </dt>
  <dd>
    Arthur Dent
  </dd>
  <dt>
    Username:
  </dt>
  <dd>
    panickedBrit
  </dd>
  <dt>
    Email:
  </dt>
  <dd>
    DentedFord42@yahoo.com
  </dd>
</dl>
`;
